ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/Configuration.scala

```diff
@@ -11,15 +11,18 @@ import scala.util.Try
   * Emitted in [[com.daml.ledger.participant.state.v1.Update.ConfigurationChanged]].
   *
   * @param generation The configuration generation. Monotonically increasing.
-  * @param timeModel The time model of the ledger. Specifying the time-to-live bounds for Ledger API commands.
-  * @param maxDeduplicationTime The maximum time window during which commands can be deduplicated.
+  * @param timeModel The time model of the ledger.
+  * @param deduplicationPeriodLengthGuarantee A lower bound on the length of the deduplication period for command submissions
+  *                                           that the WriteService provides.
   */
 final case class Configuration(
     generation: Long,
     timeModel: TimeModel,
-    maxDeduplicationTime: Duration,
+    deduplicationPeriodLengthGuarantee: Duration,
```

Comment:
The interaction between this field and the `deduplicationPeriod` on the `SubmitterInfo` is not clear to me. Can you clarify?

Reply:
It plays the same role as `maxDeduplicationTime` in the old model. If the write service reports a guaranteed deduplication period length of 1 day, then it shouldn't complain about the deduplication period being too long if it receives a command that uses a period of up to one 1 (subject to the usual clock skew provisions).

Reply:
Leaving comment as FYI.